documentation/getting-started.rst

```diff
@@ -99,20 +99,50 @@ FlexMeasures is a web-based platform, so we need a user account:
 
 .. code-block::
 
-   flexmeasures new-user --username <your-username> --email <your-email-address> --roles=admin
+   flexmeasures add user --username <your-username> --email <your-email-address> --roles=admin
 
 
 * This will ask you to set a password for the user.
 * Giving the first user the ``admin`` role is probably what you want.
 
+
 Add structure
 ^^^^^^^^^^^^^
 
-Populate the database with some standard energy asset types:
+Populate the database with some standard energy asset types, weather sensor types and a dummy market:
```

Comment:
It might change later, yes. Can be simpler in any case.

At this time, types and a little standard data (dummy data source, dummy market) fall under this when creating this data. When deleting, it really means everything but time series data, so also assets, markets, sensors, users and roles.